docs/DeveloperGuide.md

```diff
@@ -1,13 +1,39 @@
 # Developer Guide
 
+* Table of Contents
```

Comment:
It seems that Table of Contents is left blank here. It might be a good practice to work on ToC from the start stage.